crates/sui-types/src/sui_system_state/sui_system_state_inner_v1.rs

```diff
@@ -8,10 +8,10 @@ use crate::committee::{Committee, CommitteeWithNetworkMetadata, NetworkMetadata,
 use crate::crypto::verify_proof_of_possession;
 use crate::crypto::AuthorityPublicKeyBytes;
 use crate::id::ID;
+use crate::multiaddr::Multiaddr;
```

Comment:
Cool. There is a commented-out code in this file that calls `multiaddr_to_anemo_address` (which we need to uncomment asap), could you update the comment to use the new API?

Reply:
will do in a follow up